bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java

```diff
@@ -75,24 +74,15 @@ public void setup() throws IOException {
       return;
     }
 
-    Configuration config = HBaseConfiguration.create();
-
-    String extraResources = System.getProperty("bigtable.test.extra.resources");
-    if (extraResources == null) {
-      Assert.fail("Please set bigtable.test.extra.resources");
-    }
-
-    InputStream resourceStream =
-        TestClusterAPI.class.getClassLoader().getResourceAsStream(extraResources);
-    if (resourceStream == null) {
-      Assert.fail(extraResources + " does not exist");
+    Configuration configuration = new Configuration();
+    for (Entry<Object, Object> entry : System.getProperties().entrySet()) {
```

Comment:
The ultimate goal here is to separate the configuration in BigtableOptionsFactory to not rely on a Configuration object, which is HBase specific.  The change I have in mind will be a bit more intrusive than just changing a Unit test.  Are you ok with me making a much bigger PR out of this change?